src/browser/ui/dom/HTMLDOMPropertyConfig.js

```diff
@@ -104,6 +104,7 @@ var HTMLDOMPropertyConfig = {
     httpEquiv: null,
     icon: null,
     id: MUST_USE_PROPERTY,
+    inputMode: null,
```

Comment:
Looks like it's not super well supported, might be safer to go with attributes for now. The important thing to test is that updates work. Initial render will generate the right HTML either way but updates use DOM APIs.

I tested this a bit:

Firefox seems to support it either way:
![screen shot 2015-05-18 at 10 07 52 pm](https://cloud.githubusercontent.com/assets/8445/7695871/db696efe-fdaa-11e4-81cc-cbbbbe904e46.png)

Chrome not so much:
![screen shot 2015-05-18 at 10 08 10 pm](https://cloud.githubusercontent.com/assets/8445/7695872/e3bc590e-fdaa-11e4-9605-00abf5a3559d.png)